pulsar/go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/project-flogo/messaging-contrib/pulsar
 go 1.12
 
 require (
-	github.com/apache/pulsar-client-go v0.2.0
+	github.com/apache/pulsar-client-go v0.3.0
 	github.com/apache/pulsar/pulsar-function-go v0.0.0-20200819065839-6cae4afac3c3
 	github.com/project-flogo/core v0.10.1
+	github.com/sirupsen/logrus v1.4.2
```

Comment:
We wrapper Zap as pulsar client log.  I think we can remove logrus now.

Reply:
go.mod and sum updated with removing logrus dependency. Done